lib/brightbox-cli/commands/login.rb

```diff
@@ -0,0 +1,53 @@
+module Brightbox
+  command [:login] do |cmd|
+    cmd.desc I18n.t("login.desc")
+    cmd.arg_name "email"
+
+    cmd.desc "password, if not specified you will be prompted"
+    cmd.flag [:p, "password"]
+
+    cmd.desc "default account"
+    cmd.flag [:a, :account]
```

Comment:
tempted to make this explicitly --default-account (and drop the short form -a) to avoid any confusion with the global option for specifying account.